src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -142,8 +142,6 @@ public class Measure {
       message = "Version ID is required.")
   private String versionId;
   private String cmsId;
-  
-  private ReviewMetaData reviewMetaData = new ReviewMetaData();
```

Comment:
I don't think we would want to remove this. This is still going to be there for already versioned measures. we could just annotate it with @Deprecated

Reply:
I added the deprecated flag to prevent issue with already versioned measures, and re added the ReviewMetaData.java